lib/github.js

```diff
@@ -39,6 +39,13 @@ export default {
   createStatus: async (owner, repo, head_sha, entries) => {
     const github = await getRestClient();
 
+    const downloadLinks = entries.map(entry => `- [${entry.artifactId}](${entry.url})`).join("\n")
+    const pluginManager = entries.map(entry => `${entry.artifactId}:incrementals;${entry.groupId};${entry.version}&#60;/version>&#xA;&#60;/dependency>`).join("&#xA;")
```

Comment:
this seems to do for all artifacts when it will only work for plugins, is that an issue?

Reply:
I've took the easy way of rendering all artifacts.
How can I determine if it's a plugin or not?

Reply:
Addressed in last two commits.